apps/catalog/catalog/src/app/dashboard/sales/shell/view/view.component.html

```diff
@@ -1,10 +1,10 @@
 <ng-container *ngIf="sale$ | async as sale; else loading">
   <ng-container *ngIf="sale.negotiation as negotiation">
     <ng-container *ngIf="negotiation.status === 'pending' && negotiation.createdByOrg !== orgId">
-      
+
       <header fxLayout fxLayoutAlign="start center" fxLayoutGap="16px" *ngIf="sale.negotiation as negotiation">
         <ng-container *ngIf="negotiation.price else noPrice">
-          <button mat-stroked-button color="primary" (click)="accept(sale.id, sale.movie)">
+          <button mat-stroked-button color="primary" (click)="accept(sale.id, negotiation.id, sale.movie)">
```

Comment:
for coherence switch negotiationId & contractId :
```suggestion
          <button mat-stroked-button color="primary" (click)="accept(negotiation.id, sale.id, sale.movie)">
```